src/pages/docs/creators/ray/outbound.md

```diff
@@ -0,0 +1,206 @@
+https://ipfs.io/ipfs/bafkreib7pg5xwq23auzbmuo257jxjtogqhoan6vgly3u4obtpoemubdn5i -> e
+https://ipfs.io/ipfs/bafkreidcyzvhuxoxbqyumymampbujzjr43kllhrxtaeeiphjmkz2xvr4li -> m
+https://ipfs.io/ipfs/bafkreibwvht7dsk3ql73tf2d4dc4jtuv3a6juqykvrm7qtxtzp5lmfcqna ->
+https://ipfs.io/ipfs/bafkreihwavxppciktfeuynevdal4f3kp2nqivbei54fon4vpvsj625ufjy -> rocks
+https://ipfs.io/ipfs/bafkreif3oielzg25pqcpci3kqkqasos6gp2aii6vxkguezxxbewdxjb3mi -> !
+Ray on Golem rocks!
+```
+
+### Running the example on Ray on Golem cluster
+
+Now that we have the app ready, we need to start the cluster to execute it.
+
+The first step is adding the `"https://ipfs.io"` to `outbound_urls` property in the cluster yaml.
+
+You can either update your cluster yaml or download one from [the repository](https://github.com/golemfactory/golem-ray/blob/main/examples/ipfs-golem-cluster.yaml).
+The linked config is a version of the [example Ray on Golem cluster yaml](https://github.com/golemfactory/golem-ray/blob/main/golem-cluster.yaml) with:
+```yaml
+outbound_urls: ["https://pypi.dev.golem.network", "https://ipfs.io"]
+```
+
+The second step is `ray up ipfs-golem-cluster.yaml`, followed by the execution of the script:
+```bash
+ray submit golem-cluster.yaml simple-outbound-ipfs.py
+```
+```
+Ray On Golem
+  Not starting webserver, as it's already running
+Fetched IP: 192.168.0.3
+Fetched IP: 192.168.0.3
+2023-12-06 12:02:30,637	INFO worker.py:1458 -- Connecting to existing Ray cluster at address: 192.168.0.3:6379...
+2023-12-06 12:02:30,667	INFO worker.py:1633 -- Connected to Ray cluster. View the dashboard at 192.168.0.3:8265
+https://ipfs.io/ipfs/bafkreiemev2isidd7gk7352wxtqh6rwbuumt4vgnkkbx5wi6giaizt2bvq -> R
+https://ipfs.io/ipfs/bafkreigks6arfsq3xxfpvqrrwonchxcnu6do76auprhhfomao6c273sixm -> a
+https://ipfs.io/ipfs/bafkreifb7tsdmocu76eiz72lrz4hlvqayjuchecbfkgppgzx2cyrcsfq7i -> y
+https://ipfs.io/ipfs/bafkreiaplag3id7ckuzs6eiwca3skuheddup7p7espat5omqv6r7m2byhi ->  on
+https://ipfs.io/ipfs/bafkreibthyfb4j4blugo5zk4i476hxet2vwghy564kz3jlxi53lnoamrum -> G
+https://ipfs.io/ipfs/bafkreidfy5gbljugdb53no7zswhust6gxaagqa2kmwnjvvcjsgyiywhs2i -> o
+https://ipfs.io/ipfs/bafkreifmvsdmbzqjzkig6yzlbyw2ztfsw56sfmdcd4qox3hbusbvxe7w6a -> l
+https://ipfs.io/ipfs/bafkreib7pg5xwq23auzbmuo257jxjtogqhoan6vgly3u4obtpoemubdn5i -> e
+https://ipfs.io/ipfs/bafkreidcyzvhuxoxbqyumymampbujzjr43kllhrxtaeeiphjmkz2xvr4li -> m
+https://ipfs.io/ipfs/bafkreibwvht7dsk3ql73tf2d4dc4jtuv3a6juqykvrm7qtxtzp5lmfcqna ->
+https://ipfs.io/ipfs/bafkreihwavxppciktfeuynevdal4f3kp2nqivbei54fon4vpvsj625ufjy -> rocks
+https://ipfs.io/ipfs/bafkreif3oielzg25pqcpci3kqkqasos6gp2aii6vxkguezxxbewdxjb3mi -> !
+Ray on Golem rocks!
+Shared connection to 192.168.0.3 closed.
+```
+
+Failing outbound calls would get logged to the console and retried. 
+Note that it may happen that one or more calls would hang for a couple of minutes eventually displaying `[Errno -3] Temporary failure in name resolution)`. 
```

Comment:
```suggestion
Note that it may sometimes happen that one or more requests hangs for a couple of minutes, eventually displaying `[Errno -3] Temporary failure in name resolution)`.
```